client/tools/mgr-push/Makefile.python

```diff
@@ -2,18 +2,46 @@ THIS_MAKEFILE := $(realpath $(lastword $(MAKEFILE_LIST)))
 CURRENT_DIR := $(dir $(THIS_MAKEFILE))
 include $(CURRENT_DIR)../../../rel-eng/Makefile.python
 
+MGR-PUSH_ROOT_DIR = /manager/client/tools/mgr-push
+RHNPUSH_PACKET_DIR = $(MGR-PUSH_ROOT_DIR)/rhnpush
+
 # Docker tests variables
+DOCKER_REF            = $(DOCKER_REGISTRY)/$(DOCKER_IMAGE)
 DOCKER_CONTAINER_BASE = systemsmanagement/uyuni/master/docker/containers/uyuni-master
 DOCKER_REGISTRY       = registry.opensuse.org
-DOCKER_RUN_EXPORT     = "PYTHONPATH=$PYTHONPATH"
+DOCKER_RUN_EXPORT     = "PYTHONPATH=/manager/client/tools/mgr-push/rhnpush:/manager/python/:/manager/client/tools/mgr-push:/manager/client/rhel/spacewalk-client-tools/src/"
 DOCKER_VOLUMES        = -v "$(CURDIR)/../../../:/manager"
 
+define setup_structure
+    mv $(MGR-PUSH_ROOT_DIR)/rhnpush $(MGR-PUSH_ROOT_DIR)/rhnpush_wrapper
+    mkdir $(RHNPUSH_PACKET_DIR)
+    cd $(MGR-PUSH_ROOT_DIR) && mv archive.py connection.py rhnpush_cache.py rhnpush_config.py rhnpush_confmanager.py rhnpush_main.py rhnpush_v2.py rpm2mpm.py uploadLib.py utils.py $(RHNPUSH_PACKET_DIR)
+    touch $(RHNPUSH_PACKET_DIR)/__init__.py
```

Comment:
Here, instead of `mv` I would use `cp` to some testing path, to avoid messing the enviroment in the host of the container while the tests are running.

Reply:
You're right, that's better.